tasks/client.yml

```diff
@@ -1,16 +1,29 @@
 ---
-# TODO Install clients:
-## kubectl
-## aws-iam-authenticator
 
-- name: "Get kubectl path"
+- name: Get kubectl path
   shell: "which kubectl || echo 'not found'"
   register: kubectl_path
 
+- name: Get last stable version of kubectl
+  get_url:
+    url: https://storage.googleapis.com/kubernetes-release/release/stable.txt
+    dest: /tmp/kubectl_version
+  when: '"not found" in kubectl_path.stdout'
+
+- name: Get kubectl version
+  shell: "cat /tmp/kubectl_version"
+  register: kubectl_version  
+  when: '"not found" in kubectl_path.stdout'
+
+- name: Log kubectl version
+  debug:
+    var: kubectl_version.stdout
+  when: '"not found" in kubectl_path.stdout'
+   
 - name: Download kubectl
   get_url:
-    url: https://amazon-eks.s3-us-west-2.amazonaws.com/1.11.5/2018-12-06/bin/linux/amd64/kubectl
-    dest: ~/bin/kubectl
+    url: https://storage.googleapis.com/kubernetes-release/release/{{ kubectl_version.stdout }}/bin/linux/amd64/kubectl
+    dest: /usr/local/bin/kubectl
     mode: 0775
   when: '"not found" in kubectl_path.stdout'
```

Comment:
need to remove this section (cant suggest in github since its not part of the commit)